app/views/hyrax/base/_share_with.html.erb

```diff
@@ -1,14 +1,6 @@
-<p>You may grant &quot;View/Download&quot; or &quot;Edit&quot; access for specific users and/or groups to files.
-  Enter a valid <%= t('hyrax.account_name') %>, one at a time, select the access level for that user and click
-  <button class='btn btn-xs btn-inverse' onclick='return false;'><i class='glyphicon glyphicon-plus'></i>Add</button>.
-</p>
-
+<p /><p><%= t('.share_with_html', account_name: t('hyrax.account_name')) %></p>
```

Comment:
What's going on with the empty `<p />` tag here? Can that be removed?

Reply:
I did it to improve the spacing under the Share With heading below. 

These views need more extensive CSS work to get them consistent with other views, and there wasn't a class here to modify so I did it the easy way. I'll just take it out and leave that fix for someone else to do the right way.

![screen shot 2018-07-12 at 10 05 08 am](https://user-images.githubusercontent.com/17851674/42638864-afdc3ada-85bc-11e8-99b1-f18357d4f2ea.png)

Reply:
Thanks. that makes sense to me.